conans/client/rest/uploader_downloader.py

```diff
@@ -174,9 +178,26 @@ def progress_units(progress, total):
     return int(50 * progress / total)
 
 
-def print_progress(output, units):
+def human_readable_progress(bytes_transferred, total_bytes):
```

Comment:
There is a ``human_size`` function that shares most of the logic of this one, I think it would be good to re-use it if possible: https://github.com/conan-io/conan/blob/develop/conans/tools.py#L206